scripts/zones/Bastok_Markets_[S]/npcs/Kevan.lua

```diff
@@ -0,0 +1,45 @@
+-----------------------------------
+-- Area: Bastok Markets [S]
+--  NPC: Kevan
+-- !pos -308.590 -012.000 -094.227 189
+-----------------------------------
+local ID = require("scripts/zones/Bastok_Markets_[S]/IDs")
+require("scripts/globals/keyitems")
+require("scripts/globals/settings")
+-----------------------------------
+local entity = {}
+
+entity.onTrade = function(player, npc, trade)
+    local itemID = trade:getItemId()
+    local itemKeyItemMapping = {
+        [2686] = xi.ki.THE_WORDS_OF_DONHU_I,
+        [2687] = xi.ki.THE_WORDS_OF_DONHU_II,
+        [2688] = xi.ki.THE_WORDS_OF_DONHU_III,
+        [2689] = xi.ki.THE_WORDS_OF_DONHU_IV,
+        [2690] = xi.ki.THE_WORDS_OF_DONHU_V,
+        [2691] = xi.ki.THE_WORDS_OF_DONHU_VI,
+        [2692] = xi.ki.THE_WORDS_OF_DONHU_VII,
+        [2693] = xi.ki.THE_WORDS_OF_DONHU_VIII
```

Comment:
This items would regularly need to be added to scripts/globals/items.lua

However, since we are moving that file location, I think we can wait until that gets moved.